contrib/examples/apiserver/serviceclass.yaml

```diff
@@ -3,8 +3,8 @@ kind: ServiceClass
 metadata:
   name: d35b55b2-b1fd-4123-8045-5b9c619cb629
 spec:
-  brokerName: test-broker
+  servicebrokerName: test-broker
```

Comment:
given we will probably have global and local brokers, I wonder if this should be a ref instead of just a name